src/content_script.js

```diff
@@ -1,74 +1,73 @@
-import $ from "jquery";
+/* eslint-disable require-jsdoc */

```

Comment:
I figure this script isn't long for this world in this format anyway so documenting it by hand wouldn't be useful.